### ISSUE TITLE
Brighter symbols(overlay maybe), Horizon animation.

### ISSUE THREAD

**Issue by MrJake222** [MrJake222/AUNIS#45]
Issue by _skrabyz_ from Curse

1) Please make the **symbols** on the dhd **brighter**. I spend like 5 minutes dialing an address because I can't see where any of the symbols are, and I have to check the entire thing with my face pressed against my screen.

2) Consider **animating**  the event horizon?

**Comment by skrabyz** [comment]
I also would like to suggest that the Stargate should be audible from much farther away. As of right now, you must be within approximately 10 blocks of the gate to be able to hear it (just a guess based on observation), but considering the gate is 11 blocks wide, it feels off. I propose that 20-25 blocks away for the maximum hearing range would work better aesthetically.

**Comment by MrJake222** [comment]
Yes, It should. But back then I didn't find a way to do so. I should add this to the #known-issues on Discord.
